api/server.ts

```
@@ -1,16 +1,56 @@
 import { initializeMcpApiHandler } from "../lib/mcp-api-handler";
+import registerTools from "./tools";
 
 const handler = initializeMcpApiHandler(
-  (server) => {
-    // Add more tools, resources, and prompts here
-    server.tool("echo", { message: z.string() }, async ({ message }) => ({
-      content: [{ type: "text", text: `Tool echo: ${message}` }],
-    }));
+  (server, apiKey) => {
+    // Register Meeting BaaS SDK tools with the provided API key
+    server = registerTools(server, apiKey);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)